### ISSUE TITLE
Change naming convention

### ISSUE THREAD

**Issue by oguzhanunlu** [snowplow-incubator/snowplow-event-generator#16]
Issues with current convention:

1 - It prevents running event-generator in multiple instances for the same target
2 - It doesn't allow more than 9999 files, as it formats based on 4 digits

**Comment by istreeter** [comment]
Hey @oguzhanunlu I agree with point 1, but I cannot re-produce point 2.

The files are indeed based on 4 digits to start with, but the 10,000th file is called `raw_10000.gz` and the 100,000th file is called `raw_100000.gz`.

**Comment by oguzhanunlu** [comment]
You're right @istreeter , that bug was pre-0.2.0, it is fixed [here](https://github.com/snowplow-incubator/snowplow-event-generator/commit/9aa1a45698e91e7a16d526250d01d6a860b01cc4#diff-f2a13456fd5c6dd0ab10009ddb561376bc23c90f4b906cfd22e1bc68207740d1R83) in 0.2.0 release. Pre-0.2.0 it'd lead to `java.lang.IllegalArgumentException: count is negative` when num of files exceeds 9999.

It is a nit but let me add that the updated [pad](https://github.com/snowplow-incubator/snowplow-event-generator/blob/main/sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/RotatingSink.scala#L96) method supports 4 digits. i.e.

```scala
scala> def pad(idx: Int): String = f"$idx%04d"
pad: (idx: Int)String

scala> pad(1)
res3: String = 0001

scala> pad(10)
res4: String = 0010

scala> pad(10000)
res7: String = 10000
```

e.g. having `raw_10000.gz` and `raw_0001.gz` doesn't look consistent
